### PR TITLE
faq: add a summary page for all TiDB FAQs

### PR DESCRIPTION
**Pull request by TomShawn**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11089 https://github.com/pingcap/docs-cn/pull/11100
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: faq/faq-overview.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+summary: Summarizes frequently asked questions (FAQs) about TiDB.
+---
+
+# TiDB FAQ Summary
+
+This document summarizes frequently asked questions (FAQs) about TiDB.
+
+|   Category     |           Releted documents          |
+|   :-------   |   :-------------------   |
+|   TiDB architecture and its working principles    |       [TiDB architecture FAQ](/faq/tidb-faq.md)   |
```
Comment (TomShawn) [inline faq/faq-overview.md:12]:
```suggestion
|   TiDB architecture and its working principles    |       [TiDB architecture FAQs](/faq/tidb-faq.md)   |
```

File: faq/faq-overview.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+summary: Summarizes frequently asked questions (FAQs) about TiDB.
+---
+
+# TiDB FAQ Summary
+
+This document summarizes frequently asked questions (FAQs) about TiDB.
+
+|   Category     |           Releted documents          |
+|   :-------   |   :-------------------   |
+|   TiDB architecture and its working principles    |       [TiDB architecture FAQs](/faq/tidb-faq.md)   |
```
Comment (lilin90) [inline faq/faq-overview.md:12]:
```suggestion
|   TiDB architecture and principles    |       [TiDB Architecture FAQs](/faq/tidb-faq.md)   |
```

File: faq/faq-overview.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+|  Data migration      | <ul><li>[Data migration FAQs](/faq/migration-tidb-faq.md)</li><li>Data import<ul><li>[TiDB Lightning FAQs](/tidb-lightning/tidb-lightning-faq.md)</li><li>[DM FAQs](/dm/dm-faq.md)</li></ul></li><li>Incremental data replication<ul><li>[TiCDC FAQs](/ticdc/ticdc-faq.md)</li><li>[TiDB Binlog FAQs](/tidb-binlog/tidb-binlog-faq.md)</li></ul></li></ul>  |
+|  Data backup and restore   |    [Backup & Restore FAQs](/br/backup-and-restore-faq.md)   |
+|  SQL operations  |   [SQL FAQs](/faq/sql-faq.md)  |
+|  Cluster upgrade       |   [TiDB upgrade FAQs](/faq/upgrade-faq.md) |
+| Cluster management  |  [Cluster management FAQs](/faq/manage-cluster-faq.md)  |
+| Monitor and alert  |  <ul><li>[Monitoring FAQs](/faq/monitor-faq.md)</li><li>[TiDB Dashboard FAQs](/dashboard/dashboard-faq.md)</li><li>[TiDB cluster alert rules](/alert-rules.md)</li></ul>  |
+| High availability and high reliability    |   <ul><li>[High availabilty FAQs](/faq/high-availability-faq.md)</li><li>[High reliability FAQs](/faq/high-reliability-faq.md)</li></ul>   |
+| Common error codes      |  [Error codes and troubleshooting](/error-codes.md) |
```
Comment (lilin90) [inline faq/faq-overview.md:21]:
```suggestion
|  Data migration      | <ul><li>[Data Migration FAQs](/faq/migration-tidb-faq.md)</li><li>Data import<ul><li>[TiDB Lightning FAQs](/tidb-lightning/tidb-lightning-faq.md)</li><li>[DM FAQs](/dm/dm-faq.md)</li></ul></li><li>Incremental Data Replication<ul><li>[TiCDC FAQs](/ticdc/ticdc-faq.md)</li><li>[TiDB Binlog FAQs](/tidb-binlog/tidb-binlog-faq.md)</li></ul></li></ul>  |
|  Data backup and restore   |    [Backup & Restore FAQs](/br/backup-and-restore-faq.md)   |
|  SQL operations  |   [SQL FAQs](/faq/sql-faq.md)  |
|  Cluster upgrade       |   [TiDB Upgrade FAQs](/faq/upgrade-faq.md) |
| Cluster management  |  [Cluster Management FAQs](/faq/manage-cluster-faq.md)  |
| Monitor and alert  |  <ul><li>[Monitoring FAQs](/faq/monitor-faq.md)</li><li>[TiDB Dashboard FAQs](/dashboard/dashboard-faq.md)</li><li>[TiDB Cluster Alert Rules](/alert-rules.md)</li></ul>  |
| High availability and high reliability    |   <ul><li>[High Availability FAQs](/faq/high-availability-faq.md)</li><li>[High Reliability FAQs](/faq/high-reliability-faq.md)</li></ul>   |
| Common error codes      |  [Error Codes and Troubleshooting](/error-codes.md) |
```

File: faq/faq-overview.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+# TiDB FAQ Summary
+
+This document summarizes frequently asked questions (FAQs) about TiDB.
+
+|   Category     |           Releted documents          |
+|   :-------   |   :-------------------   |
+|   TiDB architecture and its working principles    |       [TiDB architecture FAQs](/faq/tidb-faq.md)   |
+|   Deployment           |     <ul><li>[Deployment FAQs](/faq/deploy-and-maintain-faq.md)</li><li>[TiUP FAQs](/tiup/tiup-faq.md)</li></ul> |
```
Comment (lilin90) [inline faq/faq-overview.md:13]: How about adding a [TiDB in Kubernetes FAQs](https://docs.pingcap.com/tidb-in-kubernetes/dev/faq) here?

File: faq/faq-overview.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+# TiDB FAQ Summary
+
+This document summarizes frequently asked questions (FAQs) about TiDB.
+
+|   Category     |           Releted documents          |
+|   :-------   |   :-------------------   |
+|   TiDB architecture and principles    |       [TiDB Architecture FAQs](/faq/tidb-faq.md)   |
+|   Deployment           |     <ul><li>[Deployment FAQs](/faq/deploy-and-maintain-faq.md)</li><li>[TiUP FAQs](/tiup/tiup-faq.md)</li><li>[TiDB in Kubernetes FAQs](https://docs.pingcap.com/tidb-in-kubernetes/stable/faq)</li></ul> |
+|  Data migration      | <ul><li>[Data Migration FAQs](/faq/migration-tidb-faq.md)</li><li>Data import<ul><li>[TiDB Lightning FAQs](/tidb-lightning/tidb-lightning-faq.md)</li><li>[DM FAQs](/dm/dm-faq.md)</li></ul></li><li>Incremental Data Replication<ul><li>[TiCDC FAQs](/ticdc/ticdc-faq.md)</li><li>[TiDB Binlog FAQs](/tidb-binlog/tidb-binlog-faq.md)</li></ul></li></ul>  |
```
Comment (lilin90) [inline faq/faq-overview.md:14]:
```suggestion
|  Data migration      | <ul><li>[Data Migration FAQs](/faq/migration-tidb-faq.md)</li><li>Data import<ul><li>[TiDB Lightning FAQs](/tidb-lightning/tidb-lightning-faq.md)</li><li>[DM FAQs](/dm/dm-faq.md)</li></ul></li><li>Incremental data replication<ul><li>[TiCDC FAQs](/ticdc/ticdc-faq.md)</li><li>[TiDB Binlog FAQs](/tidb-binlog/tidb-binlog-faq.md)</li></ul></li></ul>  |
```

File: faq/faq-overview.md (hunk 4 above)
Comment (TomShawn) [reply faq/faq-overview.md]: Added. Chinese added in https://github.com/pingcap/docs-cn/pull/11100.

File: faq/faq-overview.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+---
+title: TiDB FAQ Summary
+summary: Summarizes frequently asked questions (FAQs) about TiDB.
+---
+
+# TiDB FAQ Summary
+
+This document summarizes frequently asked questions (FAQs) about TiDB.
+
+|   Category     |           Releted documents          |
```
Comment (lilin90) [inline faq/faq-overview.md:10]:
```suggestion
|   Category     |           Related documents          |
```

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**